external/openglcts/modules/common/glcPixelStorageModesTests.cpp

```diff
@@ -0,0 +1,2498 @@
+/*-------------------------------------------------------------------------
+* OpenGL Conformance Test Suite
+* -----------------------------
+*
+* Copyright (c) 2020 The Khronos Group Inc.
+* Copyright (c) 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/ /*!
+* \file  glcPixelStorageModesTests.cpp
+* \brief Conformance tests for usage of pixel storage modes.
+*/ /*-------------------------------------------------------------------*/
+
+#include "stdlib.h"
+#include "tcuRenderTarget.hpp"
+#include "tcuSurface.hpp"
+#include "tcuTextureUtil.hpp"
+#include "tcuTestCase.hpp"
+#include "tcuTestLog.hpp"
+#include "tcuDefs.hpp"
+#include "tcuFloat.hpp"
+#include "gluRenderContext.hpp"
+#include "gluShaderProgram.hpp"
+#include "gluContextInfo.hpp"
+#include "gluObjectWrapper.hpp"
+#include "gluCallLogWrapper.hpp"
+#include "gluPixelTransfer.hpp"
+#include "gluTexture.hpp"
+#include "gluTextureUtil.hpp"
+#include "gluDrawUtil.hpp"
+#include "gluDefs.hpp"
+#include "sglrGLContext.hpp"
+#include "sglrContextWrapper.hpp"
+#include "sglrContextUtil.hpp"
+#include "glwFunctions.hpp"
+#include "glwEnums.hpp"
+#include "deStringUtil.hpp"
+#include "deUniquePtr.hpp"
+#include "glsTextureTestUtil.hpp"
+#include "glcPixelStorageModesTests.hpp"
+
```

Comment:
Style-wise: use tabs for indentation, curly braces must always be on a new line

Reply:
Done.